samples/ingestion/ingestion-client/.devcontainer/devcontainer.json

```diff
@@ -4,15 +4,20 @@
     "features": {
         "ghcr.io/devcontainers/features/azure-cli:1": {},
         "ghcr.io/devcontainers/features/node:1": {},
-        "ghcr.io/jlaundry/devcontainer-features/azure-functions-core-tools:1": {}
+        "ghcr.io/jlaundry/devcontainer-features/azure-functions-core-tools:1": {},
+        "ghcr.io/devcontainers-contrib/features/pre-commit:2": {},
+        "ghcr.io/rchaganti/vsc-devcontainer-features/azurebicep:1.0.5": {}
     },
+    "postCreateCommand": "pre-commit install",
     "customizations": {
         "vscode": {
             "extensions": [
                 "ms-dotnettools.csdevkit",
                 "ms-vscode.azure-account",
                 "ms-azuretools.vscode-azurefunctions",
-                "editorconfig.editorconfig"
+                "editorconfig.editorconfig",
```

Comment:
Can we add `github.vscode-pull-request-github` whilst we are here so we can review Pull Requests from vscode

Reply:
Updated